docs/deployment/getting_started_locally.md

```diff
@@ -279,7 +279,7 @@ There are cases where you would want to create a second seed cluster in your loc
 If you are on macOS, add a new IP address on your loopback device which will be necessary for the new KinD cluster that you will create. On macOS, the default loopback device is `lo0`.
 
 ```bash
-sudo ip addr add 127.0.0.2 dev lo0                                     # adding 127.0.0.2 ip to the loopback interface
+sudo ip addr add 172.18.255.2 dev lo0                                  # adding 172.18.255.2 ip to the loopback interface
```

Comment:
There are a couple of occurrences of `127.0.0.1` in this file, e.g. in this section: https://github.com/gardener/gardener/blob/master/docs/deployment/getting_started_locally.md#accessing-the-shoot-cluster